python/setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(name='snowballstemmer',
       version='1.2.0',
-      description='This package provides 18 stemmer algorithms (17 + Porter English stemmer) generated from Snowball algorithms.',
+      description='This package provides 19 stemmer algorithms (17 + Porter English stemmer) generated from Snowball algorithms.',
```

Comment:
`17` should be `18` now (I'm not convinced having an exact count here is worth the maintenance cost, but while it's here only updating one of the two numbers is unhelpful).